runtime/include/chpltypes.h

```diff
@@ -31,14 +31,12 @@
 #include <string.h>
 #include <sys/time.h> // for struct timeval
 
-#ifndef __cplusplus
 #include <complex.h>
-typedef float complex        _complex64;
-typedef double complex       _complex128;
-#else
-#include <complex>
-typedef std::complex<float>  _complex64;
-typedef std::complex<double> _complex128;
+typedef float _Complex        _complex64;
+typedef double _Complex       _complex128;
+
+#ifndef _Complex_I
+#define _Complex_I __builtin_complex(0.0,1.0)
```

Comment:
Do you know when this comes up? I would think we are doing C99 stuff? This fall-back looks GCC/clang specific to me.